styles.css

```diff
@@ -0,0 +1,209 @@
+@import url('https://fonts.cdnfonts.com/css/poppins');
+
+:root {
+  --main-bg-color: #e5e5e5;
+  --menu-bg-color: #6070ff;
+  --header-text-color: #172b4d;
+  --text-color: #344563;
+  --icon-color: #505f79;
+  --primary-color: #6070ff;
+}
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: var(--main-bg-color);
+}
+
+header {
+  display: flex;
+  align-items: center;
+  padding: 10px 24px;
+  justify-content: space-between;
+  background-color: white;
+  position: fixed;
+  width: 100%;
+}
+
+main {
+  overflow-y: auto;
+}
+
+.logo {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  text-align: center;
+  color: var(--primary-color) !important;
+}
+
+.bio {
+  height: 718px;
+  width: 100%;
+  background-color: #fff;
+  background-image: url('./assets/images/header-bg.svg');
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding: 150px 24px;
+  background-origin: 0 0;
+  border-bottom-left-radius: 20%;
+}
```

Comment:
- [ ] Good job adding the `bio section` to your web page 👏. The requirement says that you should stick as much as possible to the Figma design , I noticed that the background position is not exactly as it is in the Figma design. Kindly fix that 👍 

| Your design | Figma design |
| :---: | :---: |
|  ![Capture](https://user-images.githubusercontent.com/80458199/235126970-a1328b09-b380-4ac7-90fb-b79d38d0537b.PNG) | ![Capture3](https://user-images.githubusercontent.com/80458199/235127005-3b10424a-778a-4317-80d3-8176fd63c90f.PNG) |